src/main/java/org/jsonrql/Keywords.java

```diff
@@ -22,7 +23,8 @@
     {
         try
         {
-            KEYWORDS = new ObjectMapper().readValue(Keywords.class.getResource("keywords.json"), Keywords.class);
+            URL resource = Keywords.class.getClassLoader().getResource("keywords.json");
```

Comment:
personally I always use `final` if the variable won't change. But gosh it feels like a long time ago I was doing Java. Your thoughts welcome if you disagree!